.github/workflows/release-published.yml

```diff
@@ -222,4 +222,46 @@ jobs:
         run: mvn -B clean deploy -DskipTests=true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          TARGET_ARTIFACT_REPOSITORY: github
\ No newline at end of file
+          TARGET_ARTIFACT_REPOSITORY: github
+
+  deploy_xsd:
+      name: Upload xsds
+      needs: [ setup ]
+      runs-on: ubuntu-20.04
+      outputs:
+          tag: ${{ steps.collect-data.outputs.tag }}
+          version: ${{ needs.setup.outputs.version }}
+      steps:
+      - name: Download liquibase xsd
+        uses: actions/checkout@v3
+        with:
+          # Relative path under $GITHUB_WORKSPACE to place the repository
+          path: liquibase-core  
+          repository: "liquibase/liquibase"
+    
+      - name: Download liquibase-pro xsd
+        uses: actions/checkout@v3
+        with:
+        # Relative path under $GITHUB_WORKSPACE to place the repository
+          path: liquibase-pro
+          repository: "liquibase/liquibase-pro"
+
+      - name: Get major.minor xsd
+        run: |
+          version=${{ needs.setup.outputs.version }}
+          arr=(${version//./ })
+          xsd_version=${arr[0]}"."${arr[1]}
+          patch_version=${arr[2]}
+          echo ::set-env name=XSD_VERSION::${xsd_version}
+          echo ::set-env name=PATCH_VERSION::${patch_version}
+
+      - name: Upload to s3
+        # if patch_version is 0, only then upload to s3 as it's a new minor release 
+        if: ${{ env.PATCH_VERSION }} == 0
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.LIQUIBASEORIGIN_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.LIQUIBASEORIGIN_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: us-east-1
+        run: |
+          aws s3 sync $GITHUB_WORKSPACE/liquibase-core/src/main/resources/www.liquibase.org/xml/ns/dbchangelog/dbchangelog-${{ env.XSD_VERSION }}.xsd s3://liquibaseorg-origin --only-show-errors
```

Comment:
I don't know how the s3 sync command works, but nether seem to have a target directory listed? They go in different spots in the s3 hierarchy, is that being managed somewhere?

Reply:
I just got access to the SFTP server and it will include the full path in my next commit.

Reply:
@nvoxland is this comment resolved?

Reply:
I have added a full path for the `xsd`'s which will grab the `xsd` from the source in `liquibase` GitHub repo to the target dir in `s3` which are different for `liquibase-pro` and `liquibase`.